### PR TITLE
docs: add detail to the instructions to set up an Nx Stacks workspace

### PR DESCRIPTION
**Pull request by georgewilde**
#### 🤔 Why
		
To make is easier for users getting started with an Nx Stacks workspace.

### CODE REVIEW

File: docs/nx/nx_monorepo.md [hunk 1]
```diff
@@ -106,9 +107,11 @@ Visit the [NX Example](../stackscli/about.md) for the complete instructions/exam
 
 :::
 
-### Option 2: Create stacks workspace generator
+#### Option 2: Create stacks workspace generator
 
-You can scaffold a brand new Stacks and Nx workspace through the **@ensono-stacks/create-stacks-workspace** package.
+You can scaffold a brand-new Stacks and Nx workspace using the **@ensono-stacks/create-stacks-workspace** package.
+
+This approach is ideal if you want to use Stacks to build your frontend web applications but already have an existing infrastructure set up without Stacks.
```
Comment (Danphillipz) [inline docs/nx/nx_monorepo.md:114]: Worth adding a note to say that it's still possible to create a workspace with infrastructure through this option, however you would need to pass in the CLI arguments listed on this page: https://stacks.amido.com/docs/nx/create-stacks-workspace/ensono-stacks-create-stacks-workspace#command-line-arguments

Comment (georgewilde) [reply docs/nx/nx_monorepo.md]: I've updated the description for "Option 2" now. Sorry, I squashed the commits without thinking, so the update is part of the one commit.

File: docs/nx/nx_monorepo.md [hunk 2]
```diff
@@ -78,15 +78,16 @@ yarn global add nx
...
+There are three ways to get started with building a **CSR (React)** or **SSR (Next JS)** frontend web application with Stacks, we recommend using our _[Stacks CLI](../stackscli/about.md)_ for the best experience!
 
 :::note
 
 CSR support is currently limited however we are actively working on this!
 
 :::
 
-### Option 1: Stacks CLI
+#### Option 1: Stacks CLI
```
Comment (Danphillipz) [inline docs/nx/nx_monorepo.md:89]:
```suggestion
#### Option 1: Stacks CLI

```

**Pull request merged**